layouts/partials/head.html

```diff
@@ -52,4 +52,50 @@
     navigator.serviceWorker.register('/sw.js', { scope: '/' })
   }
   </script>
+  {{ if .IsPage }}
```

Comment:
I don't think all this is needed, Hugo has a embedded template for it
https://www.continuoustestingmeetup.com/events/2019-05-23/

Reply:
I just tried it on the CTM site and it works
https://github.com/continuoustestingmeetup/continuoustestingmeetup.github.io/commit/dc6468d3e954cc7c9ddab66494039518415ac8ee

Reply:
Good catch, updated!